test/sequential/test-setproctitle.js

```diff
@@ -25,7 +26,8 @@ exec('ps -p ' + process.pid + ' -o args=', function(error, stdout, stderr) {
   assert.equal(stderr, '');
 
   // freebsd always add ' (procname)' to the process title
-  if (process.platform === 'freebsd') title += ' (iojs)';
+  if (process.platform === 'freebsd')
+    title += ` (${path.basename(process.execPath)})`;
```

Comment:
This will append `.exe` on Windows. I say just change it to `node`or do something [like this](https://github.com/silverwind/detect-engine/blob/master/index.js).